components/BookList/BookList.js

```diff
@@ -30,21 +12,31 @@ export default function BookList({
     <StyledList>
       {books?.map((book) => (
         <li key={book.id}>
-          <StyledLink href={`/book-details/${book.id}`}>
-            <BookCard
-              title={book.title}
-              author={book.author}
-              genre={book.genre}
-              cover={book.cover}
-              booksInfo={booksInfo}
-              bookId={book.id}
-              handleToggleAlreadyRead={handleToggleAlreadyRead}
-              handleToggleBookmark={handleToggleBookmark}
-              handleToggleCurrentlyReading={handleToggleCurrentlyReading}
-            />
-          </StyledLink>
+          <BookCard
+            title={book.title}
+            author={book.author}
+            genre={book.genre}
+            cover={book.cover}
+            booksInfo={booksInfo}
+            bookId={book.id}
+            handleToggleAlreadyRead={handleToggleAlreadyRead}
+            handleToggleBookmark={handleToggleBookmark}
+            handleToggleCurrentlyReading={handleToggleCurrentlyReading}
+          />
```

Comment:
💡**Just a suggestion**
To make the code more readable you could use the destructuring syntax.

```js

// ... above the return
 const { title, author, genre, cover, bookId} = book 
           

return (
// ...
<BookCard
            title={title}
            author={author}
            genre={genre}
            cover={cover}
            booksInfo={booksInfo}
            bookId={id}
            handleToggleAlreadyRead={handleToggleAlreadyRead}
            handleToggleBookmark={handleToggleBookmark}
            handleToggleCurrentlyReading={handleToggleCurrentlyReading}
          />
          
  // ...
  )
```